src/App.js

```diff
@@ -7,17 +7,20 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-     
+
     }
   }
-  render() {
+  render = () => {
     return (
       <div className="App">
-        <Landing />
+        <Landing updateLogin={this.updateLogin}/>
         {!this.state && <Loader />}
       </div>
     );
   }
+  updateLogin = ({ name, quote, ranking }) => {
+    this.setState({ user: { name, quote, ranking } });
+  }
 }
```

Comment:
Nice! I think this will work out well for us if we ever get to doing Local Storage, because we can just pass in the user object